src/main/scala/io/iohk/ethereum/ledger/InMemoryWorldStateProxy.scala

```diff
@@ -0,0 +1,276 @@
+package io.iohk.ethereum.ledger
+
+import akka.util.ByteString
+import io.iohk.ethereum.common.SimpleMap
+import io.iohk.ethereum.crypto.kec256
+import io.iohk.ethereum.db.storage.EvmCodeStorage.{Code, CodeHash}
+import io.iohk.ethereum.db.storage.NodeStorage.{NodeEncoded, NodeHash}
+import io.iohk.ethereum.db.storage._
+import io.iohk.ethereum.domain._
+import io.iohk.ethereum.ledger.EvmCodeStorageWrappedMap._
+import io.iohk.ethereum.ledger.NodeStorageWrappedMap._
+import io.iohk.ethereum.mpt.MerklePatriciaTrie.HashFn
+import io.iohk.ethereum.mpt.{MerklePatriciaTrie, _}
+import io.iohk.ethereum.network.p2p.messages.PV63.AccountImplicits._
+import io.iohk.ethereum.rlp.RLPImplicits._
+import io.iohk.ethereum.vm.{UInt256, Storage, WorldStateProxy}
+
+object InMemoryWorldStateProxy {
+
+  val byteArrayUInt256Serializer = new ByteArrayEncoder[UInt256] {
+    override def toBytes(input: UInt256): Array[Byte] = input.bytes.toArray[Byte]
```

Comment:
as I see in UInt256.scala
Output ByteString is padded with 0's from the left side up to UInt256.Size bytes.
do we want to encode those padded zeros as well?